src/main/webapp/app/shared/util/typed-inject.ts

```diff
@@ -2,12 +2,28 @@ import { inject, useObserver } from 'mobx-react';
 import { IRootStore } from 'app/stores';
 import { MobXProviderContext } from 'mobx-react';
 import React from 'react';
+import { Props as SelectProps } from 'react-select';
 
-export const connect = (grabStoresFn: (param: IRootStore | Record<string, never>, any) => any) => {
-  return baseComponent => {
-    const component = ownProps => {
+/**
```

Comment:
Assuming we just want to remove this?

Reply:
Good catch. I'll double check that.

Reply:
The `git diff` being weird. I added types to this function, but it's counting it as a delete followed by an add.